src/h5web/guards.ts

```diff
@@ -88,11 +87,8 @@ export function isLink(entity: Entity): entity is Link {
   return entity.kind === EntityKind.Link;
 }
 
-export function isReachable(
-  link: HDF5Link
-): link is HDF5HardLink | HDF5RootLink {
-  // Only hard and root links are considered as reachable for now
-  return link.class === HDF5LinkClass.Hard || link.class === HDF5LinkClass.Root;
+export function isHardLink(link: HDF5Link): link is HDF5HardLink {
+  return link.class === HDF5LinkClass.Hard;
```

Comment:
With `HDF5RootLink` gone, the concept of a "reachable link" feels less relevant, but happy to revert the naming.

Reply:
I like the simplification. Let's keep `isHardLink` :+1: